recipes/finam/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "finam" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/finam-{{ version }}.tar.gz
+  sha256: 6adc4b3fb0bafd7115be21cc3fd90a14295e4b9602a4ff0e1627be8210924533
+  patches:
+    - skip_c_log_test_win.patch  # [win]
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - hatchling >=1.8.0
+    - hatch-vcs
+    - pip
+  run:
+    - python >=3.8
+    - numpy >=1.14.5
+    - scipy >=1.9.0
+    - pyevtk >=1.1.1
+    - pint >=0.20.0
+    - pyproj >=3.4
+    - pandas >=1.3
+    - opensimplex >=0.4
+
+test:
+  imports:
+    - finam
+  requires:
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pytest -vv tests
+
+about:
+  summary: FINAM is not a model.
+  home: https://finam.pages.ufz.de
+  doc_url: https://finam.pages.ufz.de/finam
+  dev_url: https://git.ufz.de/FINAM/finam
+  description: |
+    FINAM is an open-source component-based model coupling framework for environmental models.
+    It aims at enabling bi-directional online couplings of models for different compartments like
+    geo-, hydro-, pedo- and biosphere.
+  license: LGPL-3.0-or-later
+  license_family: LGPL
+  license_file:
+    - COPYING
+    - COPYING.LESSER
+    - LICENSE.md
```

Comment:
Are all these 3 LGPL-3.0-or-later? If not you must list them above.

Reply:
Yes they are. This file only contains a copyright statement.

Reply:
But the `COPYING.LESSER` (LGPL) file inculdes the following lines:
```
  This version of the GNU Lesser General Public License incorporates
the terms and conditions of version 3 of the GNU General Public
License, supplemented by the additional permissions listed below.
```
Which to my understanding means, that the GPL should be included with the LGPL since the LGPL only extends the GPL text.
This is also described here: https://www.gnu.org/licenses/gpl-howto.en.html

Reply:
Sure, but as far as I know some folks make one or the other. IANAL but it would be nicer to have the explicit in there.

If it is "and" you can do: `LGPL-3.0-or-later and GPL-3.0-or-later and LicenseRef-FINAM` for the metadata there.

Reply:
Will go with `LGPL-3.0-or-later and LicenseRef-FINAM` then. Mentioning the GPL again makes no sense IMHO since this the recommended way of licensing by GNU.

Reply:
@isuruf you are the license expert here. What do you say?

Reply:
BTW, we have that in many places b/c, IMO, sometimes folks to AND or OR so people can choose. See https://github.com/conda-forge/gudhi-feedstock/blob/945e81f775b5927f9db33769754ccf7bab77f2d4/recipe/meta.yaml#L64

That way all I know is LGPL-3.0-or-later, I don't have the option to use GPL-3.0-or-later alone. And, in your logic, than we can remove GPL-3.0-or-later from here.

Reply:
But there [some parts](https://github.com/GUDHI/gudhi-devel/blob/ee577bf6bc0d88e5fa9c1752b1d7700de3e4620b/src/python/gudhi/alpha_complex.pyx#L27) are really only under `GPL` license and [others](https://github.com/GUDHI/gudhi-devel/blob/ee577bf6bc0d88e5fa9c1752b1d7700de3e4620b/src/python/gudhi/hera/__init__.py#L7) are under `MIT`. So I read this as mixed licensing. (And the (L-)GPL text is missing entirely).

Reply:
And finam is not mixed licensed or dual/multi licensed. Its only LGPL.

Reply:
@ocefpaf @isuruf is this a blocker now?

Reply:
Maybe I would be fine with LGPL OR GPL (people can of course choose to follow the more restrictive license).

Reply:
`license_file` field is correct. `license` field should simply be `LGPL-3.0-or-later`.
`LICENSE.md` just has the copyright notices, so it is not a custom license, and just needs to be packaged like how it is done here.

Reply:
Thanks for clarification @isuruf. So we did it right, yay!
What about the `AUTHORS.md`? Should it be listed here as well?

Reply:
Since the AUTHORS.md is referenced in the copyright notice, I added it again.